src/services/ui/src/pages/actions/IssueRai.tsx

```diff
@@ -79,236 +73,155 @@ export const RaiIssueForm = () => {
+                rel="noopener noreferrer"
+                className="text-blue-700 hover:underline"
+              >
+                FAQ Page
+              </Link>
+            }
+            .
+          </p>
+          <br />
+          <p>
+            We accept the following file formats:{" "}
+            <strong className="bold">.docx, .jpg, .png, .pdf, .xlsx,</strong>{" "}
+            and a few others. See the full list on the{" "}
+            {
+              <Link
+                path="/faq"
+                hash="acceptable-file-formats"
+                target={FAQ_TARGET}
+                rel="noopener noreferrer"
+                className="text-blue-700 hover:underline"
+              >
+                FAQ Page
+              </Link>
+            }
+            .
+          </p>
+          <br />
+          <p>
+            <I.RequiredIndicator />
+            At least one attachment is required.
+          </p>
+        </section>
+        {attachmentList.map(({ name, label, required }) => (
           <I.FormField
+            key={name}
             control={form.control}
-            name="additionalInformation"
+            name={`attachments.${name}`}
             render={({ field }) => (
               <I.FormItem>
-                <h3 className="font-bold text-2xl font-sans">
-                  Additional Information<span className="text-red-500">*</span>
-                </h3>
-                <I.FormLabel className="font-normal">
-                  Add anything else you would like to share with the state.
+                <I.FormLabel>
+                  {label}
+                  {required ? <I.RequiredIndicator /> : ""}
                 </I.FormLabel>
-                <I.Textarea {...field} className="h-[200px] resize-none" />
-                <I.FormDescription>4,000 characters allowed</I.FormDescription>
+                <I.Upload
+                  files={field?.value ?? []}
+                  setFiles={field.onChange}
+                />
+                <I.FormMessage />
               </I.FormItem>
             )}
           />
-          {/*-------------------------------------------------------- */}
-          <FormDescriptionText />
-          {Object.keys(form.formState.errors).length !== 0 ? (
-            <Alert className="mb-6" variant="destructive">
-              Missing or malformed information. Please see errors above.
-            </Alert>
-          ) : null}
-          {form.formState.isSubmitting && (
-            <div className="p-4">
-              <LoadingSpinner />
-            </div>
+        ))}
+        {/*-------------------------------------------------------- */}
+        <I.FormField
+          control={form.control}
+          name="additionalInformation"
+          render={({ field }) => (
+            <I.FormItem>
+              <h3 className="font-bold text-2xl font-sans">
+                Additional Information<span className="text-red-500">*</span>
```

Comment:
Don't we have a requiredindicator component somewhere. I could be wrong

Reply:
We do, but this diff only exists due to the removal of higher-nesting components. This is something I addressed on my prototyping branch but didn't include in this routing-centric PR. (I know, I crammed modals in, why not the rest? Cuz. That's why. 😉)